src/components/SchemaRow/SchemaRow.tsx

```diff
@@ -25,7 +25,7 @@ export interface SchemaRowProps {
 }
 
 export const SchemaRow: React.FunctionComponent<SchemaRowProps> = ({ schemaNode, nestingLevel }) => {
-  const description = isRegularNode(schemaNode) ? schemaNode.annotations.description : null;
+  const description = isRegularNode(schemaNode) ? schemaNode.originalFragment.description : null;
```

Comment:
one should never operate on `originalFragment`. I believe we should make it private to indicate that.
`originalFragment` may come in handy in certain situations, but it's certainly not meant to be used this way.

Reply:
putting the above comment aside, this code won't work the way user expects.
Consider the following:

```json
{
  "type": "object",
  "properties": {
    "oneOf-two-objects-hidden-description": {
      "description": "Description for oneOf with two objects. (BUG)",
      "oneOf": [
        {
          "title": "oneOf-two-items-first-item",
          "properties": {
            "name": {
              "type": "string"
            },
            "lastName": {
              "type": "string"
            }
          }
        },
        {
           "description": "Woooo for oneOf with two objects. (BUG)",
"title": "oneOf-two-items-second-item",
          "properties": {
            "nameTest2": {
              "type": "string"
            },
            "lastNameTest2": {
              "type": "string"
            }
          }
        }
      ],
      "type": "object"
    }
  }
}
```

in your case, we'll always display the description placed under `oneOf`, which is incorrect.

json-schema-tree ensures the above `description` placed under `oneOf` (and any other compound keyword like `anyOf`) is inserted into each dependency of it, so you can rely on it.

```ts
  const { defaultExpandedDepth, renderRowAddon, onGoToRef, hideExamples } = useJSVOptionsContext();

  const [isExpanded, setExpanded] = React.useState<boolean>(
    !isMirroredNode(schemaNode) && nestingLevel <= defaultExpandedDepth,
  );

  const { selectedChoice, setSelectedChoice, choices } = useChoices(schemaNode);
  const typeToShow = selectedChoice.type;
  const description = isRegularNode(selectedChoice.type) ? selectedChoice.type.annotations.description : null;
```

this will do the trick.

If the `description` is empty, I'd say a user wants to hide it for some reason, and we shouldn't display anything.


![image](https://user-images.githubusercontent.com/9273484/143207306-8b40c2b1-487e-4fcb-9d57-c2cded2d96ca.png)
![image](https://user-images.githubusercontent.com/9273484/143207325-c1aaa2b2-2ee9-4037-a32d-6aeb1d1f7756.png)